entity-api-spec.yaml

```diff
@@ -2037,6 +2326,11 @@ paths:
                 type: array
                 items:
                   type: object
+              examples:
+                organarray:
+                  value: 
+                    - collection: 'Sample object 1'
+                    - collection: 'Sample object 2'
```

Comment:
The result of `GET /entities/{id}/organs` is a json array of organ objects, and not related to any `collection` key.